src/stringProcessing/relevantWords.js

```diff
@@ -193,6 +193,30 @@ function filterOnDensity( wordCombinations, wordCount, densityLowerLimit, densit
 	} );
 }
 
+/**
+ * Filters combinations based on whether they end with a specified string or not.
+ *
+ * @param {WordCombination[]} wordCombinations The array of WordCombinations to filter.
+ * @param {string} str The string the WordCombinations that need to be filtered out end with.
+ * @param {string[]} exceptions The array of strings containing exceptions to not filter.
+ * @returns {WordCombination[]} The filtered array of WordCombinations.
+ */
+function filterEndingWith( wordCombinations, str, exceptions ) {
+	wordCombinations = wordCombinations.filter( function( combination ) {
+		let combinationstr = combination.getCombination();
+		for ( let i = 0; i < exceptions.length; i++ ) {
+			if ( combinationstr.endsWith( exceptions[ i ] ) ) {
+				return true;
+			}
+		}
+		if ( combinationstr.endsWith( str ) ) {
+			return false;
+		}
+		return true;
```

Comment:
Can't it just be `return ! combinationstr.endsWith( str );` instead of lines 212-215?

Reply:
True; implemented!